thinc/api.py

```diff
@@ -0,0 +1,83 @@
+# flake8: noqa
```

Comment:
We should just add this file to the excludes in the `setup.cfg` (at least that's how I've been doing it for the other files – or maybe this solution is cleaner, not sure.)